consul/codelab/README.md

```diff
@@ -118,47 +118,71 @@ necessary:
 }
 ```
 
+Let's assume that we are creating a base-image for a team that's running a
+service on port 80, and we want this image to register itself with a service
+named `myapp` on startup. To do so, we also need to create this `myapp.json`
+file:
+
+```json  
+{
+    "service": {
+        "name": "myapp",
+        "port": 80,
+        "checks": [
+            {
+                "id": "myapp-connect",
+                "tcp": "localhost:80",
+                "interval": "30s",
+                "timeout": "1s"
+            }
+        ]
+    }
+}
+```
+
 Now to create the image...
 
 ```bash
 # create a client node we will use to capture an image, with a bootdisk
 # that won't be auto-deleted
-gcloud compute instances create consul-client \
+gcloud compute instances create consul-client-myapp \
```

Comment:
Yeah there's a number of ways to go about this, and that one is definitely better/easier to automate. But I wanted this codelab to be explicit what the requirements are for creating a base image for a single application. For the "more advanced" stage I can create a packer template that automates this process.